test/test_optimizers.py

```diff
@@ -874,12 +976,28 @@ def test_constrained_inequality_constraints(data_lorenz, params):
 @pytest.mark.parametrize(
     "params",
     [
+<<<<<<< HEAD:test/test_optimizers.py
         dict(thresholder="l1", threshold=2, expected=2.5),
         dict(thresholder="weighted_l1", thresholds=0.5 * np.ones((1, 2)), expected=1.0),
         dict(thresholder="l2", threshold=2, expected=1.5),
         dict(
             thresholder="weighted_l2", thresholds=0.5 * np.ones((1, 2)), expected=0.75
         ),
+||||||| 5c6e9fd:test/optimizers/test_optimizers.py
```

Comment:
:checkered_flag: :checkered_flag: :checkered_flag: :checkered_flag: 

This test almost deserves its own PR given the different directions we've gone with it.  Just enumerating there to come back later:
* trapping_extended adds threshold=0 cases
* I shrunk size of problem from lorenz to mini
* You removed second test case
* I added testing for specific values equal to zero

But since I shifted the actual cvxpy work to `ConsrainedSR3`, it may make sense, if the point of this is just to check that inequality constraints are added, to remove this test?